### PR TITLE
mir_licenses: add CC Public Domain licenses

### PR DESCRIPTION
**Pull request by yagee-de**

### CODE REVIEW

**Review by eckstueck — APPROVED** [review]

File: src/main/resources/mycore-classifications/mir_licenses.xml [hunk 1]
```diff
@@ -13,8 +13,29 @@
...
+      <label xml:lang="en" text="Public Domain Licenses" />
+      <label xml:lang="de" text="Public Domain Lizenzen" />
+      <label xml:lang="x-group" text="true" />
+      <category ID="cc_pdm_1.0">
+        <label xml:lang="de" text="Public Domain Mark 1.0" description="Kein Urheberrechtsschutz" />
+        <label xml:lang="en" text="Public Fomain Mark 1.0" description="No Copyright"/>
+        <label xml:lang="x-dnb-cc" text="cc:cc-pdm-1.0"/>
+        <url xlink:href="https://creativecommons.org/publicdomain/mark/1.0/" xlink:type="locator" />
+      </category>
+      <category ID="cc_0_1.0">
```
Comment (toKrause) [inline src/main/resources/mycore-classifications/mir_licenses.xml:26]: Should be `cc_zero_1.0` far backwards compatibility with the behavior in [`mods-utils.xsl`](https://github.com/MyCoRe-Org/mycore/blob/775fc0055e656fe4ba0bbb1931cbee1cf67ddf91/mycore-mods/src/main/resources/xsl/mods-utils.xsl#L312).

**Review by toKrause — CHANGES_REQUESTED** [review]

File: src/main/resources/mycore-classifications/mir_licenses.xml (hunk 1 above)
Comment (yagee-de) [reply src/main/resources/mycore-classifications/mir_licenses.xml]: renamed new categories to cc_zero_1.0 and cc_mark_1.0

**Comment by toKrause** [comment]
[MCR-2784](https://github.com/MyCoRe-Org/mycore-classifications/pull/6/commits/c3e9438361137003be2cd299ef7fbaf3e948188f) did not introduce the naming convention that requires `cc_zero_1.0` and `cc_mark_1.0`; it merely made every effort to keep the preexisting convention in order not to  unnecessarily  break things

**Review by toKrause — APPROVED** [review]

**Comment by yagee-de** [comment]
Shouldn't we use x-uri labels for CC licenses? https://wiki.creativecommons.org/wiki/Marking_Works_Technical

**Comment by kkrebs** [comment]
I already have added this categories in some MIR applications:
https://leopard.tu-braunschweig.de/api/v2/classifications/mir_licenses

perhaps we can sync this?

And yes, there is more refacturing needed. Perhaps in a second PR?

**Review by kkrebs — APPROVED** [review]

**Pull request merged**